Acd 693 (#44)
Actindo-AG/Documentation: Fulfillment/Integration/01_ManageConnections.md

> [!!User Interface Connections](../UserInterface/03a_Connections.md)

Reviewer: ```suggestion
[!!User interface Connections](../UserInterface/03a_Connections.md)
```

# Manage the connections

Establish connections via drivers to connect your fulfillers' system with the *Actindo Core1 Platform*. 

> In the following, the general settings to create, edit, enable and disable connections are described. For certain connection drivers, you have to setup special configurations. Therefore, the connections below are described separately:
> 
> - [Fiege connection](../../FiegeIntegration/Overview/01_General.md)

Reviewer: ```suggestion
In the following, the general settings to create, edit, enable and disable connections are described. For certain connection drivers, you have to setup special configurations. Therefore, the *Fiege* connection is described separately, see [Fiege connection](../../FiegeIntegration/Overview/01_General.md).
```
Da es aktuell keine andere zu beschreibende Connection gibt, können wir das ohne Aufzählungspunkte machen.

#### Prerequisites

At least one driver has been installed.

#### Procedure

*Fulfillment > Settings > Tab CONNECTIONS*

![Connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/Connections.png "[Connection]")

1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
    The *Create connection* view is displayed.

    ![Create connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/CreateConnection.png "[Create connection]")

2. Enter a name for the connection in the *Name* field.

3. Click the *Driver* drop-down list and select the desired driver. All available drivers are displayed in the list.   

    > [Info] Drivers must be purchased from the main account and then be installed separately in each sandbox or account where you want to use it.   

    The *Credentials* section is displayed below the drop-down list.

    ![Credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/Credentials.png "[Credentials]")

    > [Info] Depending on the selected driver, the fields in the *Credentials* section differ.

> 

Reviewer: ```suggestion
```
Leerzeile weg

4. Enter the required data in the fields in the *Credentials* section.

5. Click the [SAVE] button.   
    The connection will be established. The *Checking credentials...* notice is displayed. After a few seconds, the *Loading data...* notice is displayed.

    ![Checking credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/CheckingCredentials.png "[Checking credentials]")

>     The *Create connection* view is automatically closed when the connection has been established. The *Edit connection* view of the newly created connection is displayed. The *Settings* tab is selected.

Reviewer: Nur für mein Verständnis:
Gilt das für alle Connections? Oder nur Connections mit Settings? Haben alle Connections in Fulfillment Settings?

    ![Edit connection settings](../../Assets/Screenshots/Fulfillment/Settings/Connections/EditConnectionSettings.png "[Edit connection settings]")

>     > [Info] By default, all connections that have been established are first inactive and therefore must be enabled after the connection configuration, see [Enable a connection](#enable-a-connection).

Reviewer: ```suggestion
    > [Info] By default, all connections that have been established are inactive and therefore must be enabled after the connection configuration, see [Enable a connection](#enable-a-connection).
```

## Edit a connection

Edit a connection to adjust any changed credentials, the connection name or further settings. The driver selection is read-only and cannot be edited.

#### Prerequisites

At least one connection has been created, see [Create a connection](#create-a-connection).

#### Procedure

*Fulfillment > Settings > Tab CONNECTIONS*

![Connections](../../Assets/Screenshots/Fulfillment/Settings/Connections/Connections.png "[Connections]")

1. Click the connection you want to edit in the list of connections.   
    The *Edit connection* view is displayed.

>     > [Info] Depending on the driver of the connection, the *Credentials* and the *Settings* tab are displayed in the *Edit connection* view.

Reviewer: Wenn alle Connections in Fulfillment Credentials und Settings haben, kann die Info rausfliegen.

Reviewer 2: nein nicht alle haben extra settings

    ![Edit connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/EditConnectionCredentials.png "[Edit connection]")

2. Edit the desired data of the connection in the corresponding fields in the *Credentials* tab.

3. If necessary, click the *Settings* tab and edit the desired settings of the connection in the corresponding fields.

4. Click the [SAVE] button.   
    The *Checking credentials...* notice is displayed.

    ![Credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/CheckingCredentials.png "[Credentials]")

>     The *Edit connection* view is automatically closed and the *Connections* view is displayed again.

Reviewer: ```suggestion
    The *Edit connection* view is automatically closed when the changes have been saved. The *Connections* view is displayed again.
```

## Enable a connection

> A connection can only be used if it is active. Otherwise, no data is being synchronized via the connection. By default, all connections that have been established are first inactive and therefore must be enabled. If a connection has been disabled temporarily, it can be enabled it again. It is possible to enable several inactive connections at a time.

Reviewer: ```suggestion
A connection can only be used if it is active. Otherwise, no data is being synchronized via the connection. By default, all connections that have been established are inactive and therefore must be enabled. If a connection has been disabled, it can be enabled again. It is possible to enable several connections at a time.
```

#### Prerequisites

> - At least one connection has been created, see [Create a connection](#create-a-connection).

Reviewer: ```suggestion
- At least one connection has been established, see [Create a connection](#create-a-connection).
```

> - At least one connection has been created, see [Create a connection](#create-a-connection).
> - At least one connection is inactive, see [Disable a connection](#disable-a-connection).

Reviewer: ```suggestion
At least one connection is inactive, see [Disable a connection](#disable-a-connection).
```
Eine Connection kann ja nur dann inactive sein, wenn sie vorher auch erstellt worden ist. Daher kann der erste Punkt weg.

#### Procedure

*Fulfillment > Settings > Tab CONNECTIONS*

![Connections](../../Assets/Screenshots/Fulfillment/Settings/Connections/Connections.png "[Connections]")

1. Select the checkbox of the connection you want to enable in the list of connections.   
    The editing toolbar is displayed above the list of connections.

> 2. Click the [ENABLE] button in the toolbar.   

Reviewer: ```suggestion
2. Click the [ENABLE] button in the editing toolbar.   
```

    The connection has been activated.

3. If necessary, click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button to update the list of connections.   
    The status of the enabled connection in the *Status* column has switched to **Active**.

[comment]: <> (Update der Liste läuft automatisch aber es dauert ein wenig. Refresh step vorischtshalber drin lassen?)

## Disable a connection

> Once a connection has been established, it cannot be deleted. Nevertheless, the connection can be disabled if it is no longer used. It is possible to disable several active connections at a time.

Reviewer: ```suggestion
Once a connection has been established, it cannot be deleted. Nevertheless, the connection can be disabled to prevent any further data from being synchronized via the connection, for example, if it is no longer used. It is possible to disable several active connections at a time.
```

#### Prerequisites

> - At least one connection has been created, see [Create a connection](#create-a-connection).

Reviewer: ```suggestion
- At least one connection has been established, see [Create a connection](#create-a-connection).
```

> - The connection is active, see [Enable a connection](#enable-a-connection).

Reviewer: ```suggestion
- At least one connection is active, see [Enable a connection](#enable-a-connection).
```
einheitlich -> siehe enable connection

#### Procedure

*Fulfillment > Settings > Tab CONNECTIONS*

![Connections](../../Assets/Screenshots/Fulfillment/Settings/Connections/Connections.png "[Connections]")

1. Select the checkbox of the connection you want to disable in the list of connections.   
    The editing toolbar is displayed above the list of connections.

> 2. Click the [DISABLE] button in the toolbar.   

Reviewer: ```suggestion
2. Click the [DISABLE] button in the editing toolbar.   
```

    The connection has been deactivated.

3. Click the ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh) button to update the list of connections.   
    The status of the disabled connection in the *Status* column has switched to **Inactive**.

> [comment]: <> (Update der Liste läuft automatisch aber es dauert ein wenig. Refresh step vorischtshalber drin lassen?)

Reviewer: ```suggestion
```
Ja, kannst du drin lassen. So sind wir auf der sicheren Seite.
